Gemfile

```diff
@@ -19,8 +19,7 @@ gem 'friendly_id', '5.1.0'
 gem 'gds-sso', '13.0.0'
 gem 'plek', '2.0'
 
-gem 'airbrake', '~> 5.5'
-gem 'airbrake-ruby', '1.5'
+gem 'govuk_config'
```

Comment:
Interesting! This caused `0.1.0` to be installed, which is an empty gem. I guess it needs to be '~> 0.2.0`.

Reply:
Using `govuk_app_config` now, version 0.2.0